man/systemd.exec.xml

```diff
@@ -260,6 +260,30 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RootImagePolicy=</varname></term>
+        <term><varname>MountImagePolicy=</varname></term>
+        <term><varname>ExtensionImagePolicy=</varname></term>
```

Comment:
the options are plural - ie, `MountImages` and `ExtensionImages`

Reply:
hmm, but it's an "image policy" not an "images policy"? I doubt anyone would use plural in front of policies even though it might apply to multiple objects. For example, in politics, maybe some party is pursuing a policy on cars, but they'd call it "car policy", not "cars policy"?

but i am no native speaker...

Reply:
This is not a colloquial phrase though, it is an identifier referring to other identifiers, so to me it makes more sense to precisely reference the identifier